src/Modules/Payments/Application/Payers/GetPayerEmail/PayerEmailProvider.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using System.Threading.Tasks;
+using CompanyName.MyMeetings.BuildingBlocks.Application.Data;
+using Dapper;
+
+namespace CompanyName.MyMeetings.Modules.Payments.Application.Payers.GetPayerEmail
+{
+    public static class PayerEmailProvider
+    {
+        public static async Task<string> GetPayerEmail(Guid payerId, ISqlConnectionFactory sqlConnectionFactory)
+        {
+            var connection = sqlConnectionFactory.GetOpenConnection();
+
+            const string sql = "SELECT " +
+                               "[Payer].[Email], " +
```

Comment:
It won't work, there is a comma at the end, you will have a SQL syntax error.

Reply:
Yup, you're right. Fixed and tested.